lib/scan.ts

```diff
@@ -50,6 +50,16 @@ export async function scan(options: Options): Promise<PluginResponse> {
       { type: 'fileSignatures', data: filteredSignatures },
     ];
 
+    const analytics: Analytics[] = [
+      {
+        name: 'fileSignaturesAnalyticsContext',
```

Comment:
How was that name decided? Is it defined somewhere else since before? Maybe we should share a const if that's the case. Otherwise, it might be a bit verbose, depends if we have other kinds of analytics or not.

Reply:
It's already available in our data flow.

Reply:
It's clear enough as it holds the context of what is actually happening here, and I've already spreaded it internally and is actively being used in our flows.